$NetBSD: patch-ab,v 1.3 2009/11/29 21:55:09 dholland Exp $

Avoid compiler noise when storing an integer in a pointer.

(This code will break on 32-bit platforms after 2038, but that's
somebody else's problem.)

--- libaegis/project/history/change/change2time.cc~	2008-09-24 17:48:58.000000000 -0400
+++ libaegis/project/history/change/change2time.cc	2009-11-29 15:30:43.000000000 -0500
@@ -28,13 +28,13 @@ time_t
 project_ty::change_completion_timestamp(long change_number)
 {
     assert(change2time_stp);
-    time_t result = (time_t)itab_query(change2time_stp, change_number);
+    time_t result = (time_t)(intptr_t)itab_query(change2time_stp, change_number);
     if (!result)
     {
         change::pointer cp = change_alloc(this, change_number);
         change_bind_existing(cp);
         result = ::change_completion_timestamp(cp);
-        itab_assign (change2time_stp, cp->number, (void*)result);
+        itab_assign (change2time_stp, cp->number, (void*)(intptr_t)result);
         change_free(cp);
     }
 
